<launch>
    <node name="dji_navigator" pkg="airsim_moveit_navigation" type="dji_navigator" respawn="false" output="screen" launch-prefix="gdb -ex run --args">
        <param name="/planning_plugin" value="ompl_interface/OMPLPlanner"/>
    </node>
</launch>
